### PR TITLE
7.2 | LRDOCS-6601 Elasticsearch configuration articles - SME review edits

### PR DESCRIPTION
**Pull request by lipusz**
https://issues.liferay.com/browse/LRDOCS-6601

See my changes: https://github.com/lipusz/liferay-docs/compare/5527e2b4d1a5a663d02a72e36818ce5a4e64493c...LRDOCS-6601-configuring-es-7.2

Most of these changes should go back to the 7.1 docs too. Let's don't forget it (create a task for ourselves).

Thanks,
Tibor

### CODE REVIEW

File: deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown [hunk 1]
```diff
@@ -68,23 +68,29 @@ run Elasticsearch in embedded mode in production either. Instead, run
...
+
+**Cluster Sniffing (Additional Configurations):**
+: When integrating @product@ with an Elasticsearch cluster containing multiple and different "node" [types](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-node.html#modules-node), Liferay's Elasticsearch connector may throw `NoNodeAvailableException` in the console log if no `data` node is available on the specified transport addresses. This may happen for example when only `master` type node addresses are configured. This is due to the ["cluster sniffing feature"](https://www.elastic.co/guide/en/elasticsearch/client/java-api/6.5/transport-client.html) of the Elasticsearch Transport Client that Liferay's connector is using internally and it is enabled by default:
+   >If, for instance, you initially connect to a master node, after sniffing, no further requests will go to that master node, but rather to any data nodes instead. The reason the transport client excludes non-data nodes is to avoid sending search traffic to master only nodes.
+   
+To disable cluster sniffing, add `client.transport.sniff: false` to the `additionalConfigurations` property in Liferay's Elasticsearch connector configuration.
```
Comment (rbohl) [inline deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: @lipusz Do we need `additionalConfigurations` here? Why can't we just use the dedicated `clientTransportSniff` property?

Comment (lipusz) [reply deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: My bad, I totally forgot that we have a property for that... :roll_eyes: :smile: Good catch! :+1: @rbohl

File: deployment/articles/03-installing-a-search-engine/02-elasticsearch/04-configuring-the-connector.markdown [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+    discovery.zen.ping.unicast.hosts: ["node1.ip.address", "node2.ip.address"]
+
+For example,
+
+    discovery.zen.ping.unicast.hosts: ["10.10.10.5", "10.10.10,.5:9305"]
+
+For more information on configuring an Elasticsearch cluster, see the
+documentation on 
+[Elasticsearch Index Settings](https://www.elastic.co/guide/en/elasticsearch/guide/6.5/_index_settings.html).
```
Comment (rbohl) [inline deployment/articles/03-installing-a-search-engine/02-elasticsearch/04-configuring-the-connector.markdown]: This the "guide", not the versioned reference docs we use most often. There's no 6.5 link for this.

**Comment by rbohl** [comment]
Thanks! Forwarded: https://github.com/sez11a/liferay-docs/pull/4086 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**